lib/compute-at-edge-abi/README.md

```diff
@@ -1,6 +1,6 @@
 # 🔗 compute-at-edge-abi
```

Comment:
@dora and @kpfleming - do we want to rename this and the folder?

Reply:
Oh... good question. That's going to be a breaking change for anything which consumes the content here?

Reply:
Maybe not.. it appears that the WITX files are incorporated into the crate but the directory name is not part of the crate.

Reply:
But one of the WITX files is named `compute-at-edge.witx`.